crates/rolldown/tests/test262_failures.json

```diff
@@ -0,0 +1,178 @@
+{
+  "language/module-code/parse-err-return.js": {
```

Comment:
It would be better to add some explanation about how to run specific test in the contribution gude and how this test works.

It doesn't need to be long explanation, which isn't necessary.